lib/cpp/src/thrift/transport/TSSLSocket.cpp

```diff
@@ -319,7 +319,7 @@ void TSSLSocket::checkHandshake() {
     return;
   }
   ssl_ = ctx_->createSSL();
-  SSL_set_fd(ssl_, socket_);
+  SSL_set_fd(ssl_, (int)socket_);
```

Comment:
That's reasonable; the other signedness cases these changes created should also be cleaned up elsewhere.  Thank you for reviewing!